Sources/Fuzzilli/Mutators/InputMutator.swift

```diff
@@ -28,7 +28,7 @@ public class InputMutator: BaseInstructionMutator {
         // Replace one input
         let selectedInput = Int.random(in: 0..<instr.numInputs)
         b.trace("Mutating input \(selectedInput)")
-        inouts[selectedInput] = b.randVar()
+        inouts[selectedInput] = b.randVarFromOuterScope()
```

Comment:
We don't generally want to select variables from the outer scope I think, only when the instruction is the end of a block (because then, the inner variables are still visible at this point, but shouldn't be selected as inputs)